libnmstate/ifaces/ifaces.py

```diff
@@ -194,7 +194,12 @@ def _bring_slave_up_if_not_in_desire(self):
         """
         for iface in self._ifaces.values():
             if iface.is_up and iface.is_master:
+                cur_iface = self.current_ifaces.get(iface.name)
                 for slave_name in iface.slaves:
+                    if cur_iface and slave_name in cur_iface.slaves:
+                        # Nmstate should bring up the port interface if it has
+                        # been added to the state not in all transactions
```

Comment:
Nmstate should touch the port interface which has already been included in current interface.